app/scripts/worker.js

```diff
@@ -81,7 +81,9 @@ export function workerSetPix(
   pseudocount,
   colorScale,
   ignoreUpperRight = false,
-  ignoreLowerLeft = false
+  ignoreLowerLeft = false,
+  shape = null,
+  selectedRows = null
```

Comment:
Sorry, a bit of "do as I say, not as I do here", but would you mind documenting the extra parameters you added?

Reply:
Sure thing